src/Kafka/ManageKafkaPermissions/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { sentenceCase } from "sentence-case";
+import { AclResourceType } from "./types";
+
+export const displayName = (resourceType: AclResourceType): string => {
```

Comment:
This should be a react component, and the hardcoded strings should be under translations